docs/content/en/options-reference.md

```diff
@@ -69,6 +69,19 @@ When using an object form, the properties can be:
 - `domain` (required when using `differentDomains`) - the domain name you'd like to use for that locale (including the port if used)
 - `...` - any custom property set on the object will be exposed at runtime. This can be used, for example, to define the language name for the purpose of using it in a language selector on the page.
 
+## `localeProperties`
```

Comment:
This is not an option itself so this shouldn't be listed on its own. Should be just mentioned as part of the `locales` option.

Reply:
my bad, i committed the changes, idk if it's obvious right now or not, i added it to the bottom of the locales option.